.github/workflows/gating.yaml

```diff
@@ -101,3 +110,46 @@ jobs:
         ignore: DL3041
         failure-threshold: warning
 
+  build-image:
+    name: Build Cachi2 image and run smoketest on it
+    runs-on: ubuntu-latest
+    steps:
+      - name: add checkout action...
+        uses: actions/checkout@v3
```

Comment:
How come we don't need a full clone? :thinking:  Does cachi2 in the built image have the correct version? (`cachi2 --version`)

```yaml
- uses: actions/checkout@v3
  with:
    fetch-depth: 0
```

Reply:
Fixed!